packages/position/src/index.js

```diff
@@ -1,15 +1,35 @@
 import { system } from '@styled-system/core'
 
+const defaults = {
+  space: [0, 4, 8, 16, 32, 64, 128, 256, 512],
+}
+
 const config = {
   position: true,
   zIndex: {
     property: 'zIndex',
     scale: 'zIndices',
   },
-  top: true,
-  right: true,
-  bottom: true,
-  left: true,
+  top: {
+    property: 'top',
+    scale: 'space',
+    defaultScale: defaults.space,
```

Comment:
One difference to the `@styled-system/css` package is that it also allows negative values (like margin properties) for these. I don't think you need to address that here since the `getMargin` utility isn't exported, but something to think about